### DISCUSSION TITLE
[Suggestion] Add an option to enable show full context menus & Disable lock screen

### DISCUSSION THREAD

**Discussion by aaronfilarca** [farag2/Sophia-Script-for-Windows#320] [Ideas]
Add an option to enable show full context menus & Disable lock screen

**Comment by farag2** [comment]
It's already being provided: https://github.com/farag2/Sophia-Script-for-Windows/blob/6b23fcde50d7e6841afa078c964bb41a3c9c92bc/Sophia%20Script/Sophia%20Script%20for%20Windows%2011/Sophia.ps1#L1332

> Disable lock screen

I do not advise to do it, so it won't be added.

**Comment by soredake** [comment]
Sad to see Windows 10 context menu function removed.

**Reply by farag2** [reply]
Windows 11 context menu is awesome. My personal opinion. Finally it doesn't look like a mess.

**Reply by soredake** [reply]
I can agree, but there is still a LOT of software that doesn't support new context menu, hell, even microsoft defender doesn't support it.

**Reply by soredake** [reply]
Having a choice is great, I don't think having this function in sophia script is bad.

**Reply by soredake** [reply]
New context menu is not flawless, and there are many softwares that does not support it.

Also:
https://www.outsidethebox.ms/22361/#_842
https://www.windowslatest.com/2024/11/07/windows-11s-modern-context-menu-falls-into-the-same-clutter-trap-as-windows-10/
